netplay/createroom.cpp

```diff
@@ -211,6 +231,7 @@ void CreateRoom::downloadFinished(QNetworkReply *reply)
         QStringList servers = json.keys();
         for (int i = 0; i < servers.size(); ++i)
             serverChooser->addItem(servers.at(i), json.value(servers.at(i)).toString());
+        serverChooser->addItem(QString("Custom"), QString("Custom"));
```

Comment:
I wasn't sure whether to also add this to the following code. In my testing, only downloadFinished executes, not processBroadcast, when populating the server list.

https://github.com/m64p/mupen64plus-gui/pull/67/files#diff-02ee5ce09e7d1382743485395ffd2ac8befc22c35272e87d18cbbfcdd49ffcc8R102